esi_ui/static/dashboard/esi/create-leases/create-leases-wizard.controller.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('horizon.dashboard.esi.create-leases')
+    .controller('CreateLeasesWizardController', CreateLeasesWizardController);
+
+  CreateLeasesWizardController.$inject = [
+    '$scope',
+    'createLeasesModel',
+    'horizon.dashboard.esi.create-leases.workflow',
+  ];
+
+  function CreateLeasesWizardController($scope, createLeasesModel, createLeasesWorkflow) {
+    $scope.workflow = createLeasesWorkflow;
+    $scope.model = createLeasesModel;
+    $scope.submit = createLeasesModel.submit;
+
+    $scope.model.initialize();
```

Comment:
The initialize() function doesn't exist in the model file, so that gives this error and leads to the modal not opening:

> $scope.model.initialize is not a function. (In '$scope.model.initialize()', '$scope.model.initialize' is undefined)